CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ description: |-
 # VKCS Provider's changelog
 
 #### v0.8.5 (unreleased)
+- Waiting for the active status of the subnet's dhcp ports
```

Comment:
I just don't like being specific about DHCP ports at all. Let's try to specify what problem you fix, not the way it's done. I suppose the problem is formulated as not actually configured DHCP service when subnet is returned as active?  Thus, we wait for it to become ready when creating subnet.

Reply:
And it's better to start all bugfix changes with "fix".

Reply:
done